tests/unit/utils/test_utils.py

```diff
@@ -139,3 +140,10 @@ def pattern(path):
         tmp_fname(file_path_info),
         re.IGNORECASE,
     )
+
+
+def test_relpath():
+    path = "path"
+    path_info = PathInfo(path)
+
+    assert relpath(path) == relpath(path_info)
```

Comment:
A minor thing, but we could save 3 lines here and not harm readability 🙂 
```suggestion
    assert relpath("path") == relpath(PathInfo("path"))
```